awscli/customizations/wizard/wizards/dynamodb/new-table.yml

```diff
@@ -95,6 +100,107 @@ plan:
         type: prompt
         description: Choose a customer managed CMK
         choices: existing_cmks
+        condition:
+          variable: encryption_settings
+          equals: customer_managed_cmk
+  preview:
+    shortname: Preview
+    description: Preview results
+    values:
+      preview_cli_command_value:
+        type: template
+        value: |
+          aws dynamodb create-table \
+            --table-name '{table_name}' \
+            --attribute-definitions \
+              'AttributeName={primary_key_name},AttributeType={primary_key_type}' \
+            {% if {wants_sort_key} == yes %}
+              'AttributeName={sort_key_name},AttributeType={sort_key_type}' \
+            {% endif %}
+            --key-schema \
+              'AttributeName={primary_key_name},KeyType=HASH' \
+            {% if {wants_sort_key} == yes %}
+              'AttributeName={sort_key_name},KeyType=RANGE' \
+            {% endif %}
+            {% if {encryption_settings} == aws_managed_cmk %}
+            --sse-specification 'Enabled=true,SSEType=KMS' \
+            {% endif %}
+            {% if {encryption_settings} == customer_managed_cmk %}
+            --sse-specification \
+              'Enabled=true,SSEType=KMS,KMSMasterKeyId={kms_key_id}' \
+            {% endif %}
+            {% if {capacity_mode} == PROVISIONED %}
+            --provisioned-throughput \
+               'ReadCapacityUnits={read_capacity},WriteCapacityUnits={write_capacity}'
+            {% endif %}
+            {% if {capacity_mode} == PAY_PER_REQUEST %}
+            --billing-mode PAY_PER_REQUEST
+            {% endif %}
+      preview_cfn_template_value:
+        type: template
+        value: |
+          Resources:
+            {table_name}:
+              Type: "AWS::DynamoDB::Table"
+              Properties:
+                TableName: {table_name}
+                AttributeDefinitions:
+                  - AttributeName: {primary_key_name}
+                    AttributeType: {primary_key_type}
+                  {% if {wants_sort_key} == yes %}
+                  - AttributeName: {sort_key_name}
+                    AttributeType: {sort_key_type}
+                  {% endif %}
+                KeySchema:
+                  - AttributeName: {primary_key_name}
+                    KeyType: HASH
+                  {% if {wants_sort_key} == yes %}
+                  - AttributeName: {sort_key_name}
+                    KeyType: RANGE
+                  {% endif %}
+                {% if {capacity_mode} == PROVISIONED %}
+                ProvisionedThroughput:
+                  ReadCapacityUnits: {read_capacity}
+                  WriteCapacityUnits: {write_capacity}
+                {% endif %}
+                {% if {capacity_mode} == PAY_PER_REQUEST %}
+                  BillingMode: PAY_PER_REQUEST
```

Comment:
It seems the `BillingMode` has one level of indentation too much. I was getting the following CloudFormation template:
```
Resources:
  MyTable:
    Type: "AWS::DynamoDB::Table"
    Properties:
      TableName: MyTable
      AttributeDefinitions:
        - AttributeName: Foo
          AttributeType: S
      KeySchema:
        - AttributeName: Foo
          KeyType: HASH
        BillingMode: PAY_PER_REQUEST
```
which was not valid yaml. It instead needs to be:
```
Resources:
  MyTable:
    Type: "AWS::DynamoDB::Table"
    Properties:
      TableName: MyTable
      AttributeDefinitions:
        - AttributeName: Foo
          AttributeType: S
      KeySchema:
        - AttributeName: Foo
          KeyType: HASH
      BillingMode: PAY_PER_REQUEST
```